apps/docs/pages/guides/auth/phone-login/twilio.mdx

```diff
@@ -94,7 +94,7 @@ You should now be able to see all three values you'll need to get started:
 
 ![All the credentials you'll need](/docs/img/guides/auth-twilio/6.png)
 
-Now go to the Auth > Settings page in the Supabase dashboard (https://supabase.com/dashboard/project/YOUR-PROJECT-REF/auth/settings).
+Now go to the Auth > Providers page in the Supabase dashboard and select "Phone" from the Auth Providers list (https://supabase.com/dashboard/project/YOUR-PROJECT-REF/auth/providers).
```

Comment:
```suggestion
Now go to the Auth > Providers page in the Supabase dashboard and select "Phone" from the Auth Providers list (https://supabase.com/dashboard/project/_/auth/providers).
```

Reply:
I didn't know this before, thanks for sharing the tip!